src/tree/SubscriptionTreeItem.ts

```diff
@@ -246,6 +246,14 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         return this.cache.treeMaps[groupBy] ?? {};
     }
 
+    private _setCachedChildren(childrenToSet: AzExtTreeItem[]): void {
+        // To access the private cacheChildren, go buckwild and ignore typings!!
+        const thisUnknown = this as unknown as { _cachedChildren: AzExtTreeItem[] };
```

Comment:
![image](https://c.tenor.com/LbaVZBTP-98AAAAC/frozen-elsa.gif)

It's not my fault, I have young children.

Reply:
I'm pretty sure that this is Frozen 2... Deep cut.

Reply:
🤣